styles/CSS/base.css

```diff
@@ -1,3 +1,10 @@
+/*TODO:
+1) MAKE SURE THAT CSS TAGS, ID/CLASSES ARE IN RIGHT ORDER SO THEY WORK AS THEY SHOULD.
+2) DEBUG WHY TEXT-DISPLAY: NONE IS NOT WORKING (THE LINKS/ICONS STILL UNDERLINED)
+3) CONSIDERING TODO'S ON JS FILES, ADAPT CSS TO MAKE LAYOUT CLEAN, AND EASY TO FOLLOW.
+4) GET RID OF GHOST CODE
+5) TAKE OUT ANY 'HACKY' FIXES AND UPDATE CSS TO STILL BEHAVE AS I'D LIKE*/
+
 . {
```

Comment:
For the purposes of thinking mobile-first, I would add a hamburger menu for your navigation bar. That would be a nice feature to add. I also think that displaying one article per row would be a good design to go for in order to have it look good on mobile. Then you could add a media query for desktop screens that would display your full navigation bar and you could display 2-3 posts per row. It all depends on what your content is though.